nautobot/docs/configuration/authentication/ldap.md

```diff
@@ -6,13 +6,10 @@ This guide explains how to implement LDAP authentication using an external serve
 
 ### Install System Packages
 
-!!! danger
-    FIXME(jathan): With `wheel` packages asserted, let's make doubly sure these development libraries even need to be installed anymore.
```

Comment:
Were you able to confirm that these libraries are still needed in the current installation? I see you've added them to the Dockerfile - were there failures without adding them?

Reply:
yes there definitely were

Reply:
If I may add some inputs: on Alpine, while `openldap-dev` is required to build the package, only `libldap` is required at runtime (and outside of ldap, `postgresql-dev zlib-dev jpeg-dev libffi-dev` at build time, `libpq zlib libjpeg libffi` at run time)

Reply:
@u1735067 thanks for the input, unfortunately, we don't support alpine.  I'd be curious if you are able to test using the Debian containers for runtime dependencies.